torchgeo/datasets/so2sat.py

```diff
@@ -74,18 +81,56 @@ class So2Sat(NonGeoDataset):
 
        or manually downloaded from https://dataserv.ub.tum.de/index.php/s/m1483140
        This download will likely take several hours.
-    """
 
-    filenames = {
-        "train": "training.h5",
-        "validation": "validation.h5",
-        "test": "testing.h5",
+       The version "3*" datasets can be downloaded using the following bash script:
```

Comment:
```suggestion
       The version 3 datasets can be downloaded using the following bash script:
```
Just to make things simple